### ISSUE TITLE
Please add sqlite binding

### ISSUE THREAD

**Issue by ghost** [IsaacShelton/AdeptImport#6]
This will allow working with sqlite database in Adept code.

**Comment by IsaacShelton** [comment]
Yes this is a good idea also, when I'm not busy I will look more into adding additional bindings.

Or even better, with the addition of some translation tool or automatic C support, this would come for free

I currently don't have the time to add it yet though.

Feel free to make a PR though, and I will merge
